lib/3scale/api/client.rb

```diff
@@ -24,6 +24,13 @@ def list_services
         extract(collection: 'services', entity: 'service', from: response)
       end
 
+      # @api public
+      # @param [Fixnum] id Service ID
+      def delete_service(id)
+        http_client.delete("/admin/api/services/#{id}")
```

Comment:
This does not check the response from the API?

Reply:
status code is checked, if not expected status code, error is raised.